pkg/ccl/backupccl/full_cluster_backup_restore_test.go

```diff
@@ -777,10 +776,13 @@ func TestClusterRestoreFailCleanup(t *testing.T) {
 				{"zones"},
 			},
 		)
+
+		close(blockCh)
 	})
 
 	t.Run("after offline tables", func(t *testing.T) {
-		tcRestore, sqlDBRestore, cleanupEmptyCluster := backupRestoreTestSetupEmpty(t, singleNode, tempDir, InitManualReplication, base.TestClusterArgs{})
+		tcRestore, sqlDBRestore, cleanupEmptyCluster := backupRestoreTestSetupEmpty(t, singleNode, tempDir,
```

Comment:
nit: unintended change? perhaps crlfmt was updated since this line was written. This change occurs in a few other places too.